src/traces/mesh3d/attributes.js

```diff
@@ -37,6 +37,9 @@ module.exports = {
             'jointly represent the X, Y and Z coordinates of the nth vertex.'
         ].join(' ')
     },
+    xcalendar: surfaceAtts.xcalendar,
+    ycalendar: surfaceAtts.ycalendar,
+    zcalendar: surfaceAtts.zcalendar,
```

Comment:
actually it's surface that has a problem with dates, because `colorscale` and `colorbar` don't do dates... you'll notice it's there but I had to explicitly include `surfacecolor`. We could support this too, but another time.